virocon/distributions.py

```diff
@@ -1354,31 +1355,37 @@ class VonMisesDistribution(Distribution):
     
     The distributions probability density function is given by [1]_: 
     
-    :math:`f(x) = \\frac{\\exp{k \\cos{x - \\mu}}}{2 \\pi I_0(k) \\sigma}`
+    :math:`f(x) = \\frac{\\exp{\\kappa \\cos{x - \\mu}}}{2 \\pi I_0(k) \\sigma}`
```

Comment:
There is another k in I_0(k)

Reply:
Also please add some parentheses, just from seeing the compiled equation it was not clear, that the cos is in the exponent. 
E.g. 
```
:math:`f(x) = \\frac{\\exp{(k \\cos{(x - \\mu)}})}{2 \\pi I_0(\\kappa)}`
```
(I also already removed sigma)